### PR TITLE
add label to EvaluateResult

### PR DESCRIPTION
**Pull request by yuki-mt**
## What is this PR for?

added label to `EvaluationResult` to identify which precision, recall and fvalue correspond to which labels.

## This PR includes

- related to https://github.com/rekcurd/rekcurd-python/pull/34

## What type of PR is it?

Feature/Bugfix/....

## What is the issue?

N/A

### CODE REVIEW

File: python/sklearn-digits/app.py [hunk 1]
```diff
@@ -97,8 +97,9 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
                 label_predict.append(result.label)
 
             accuracy = accuracy_score(label_gold, label_predict)
-            p_r_f = precision_recall_fscore_support(label_gold, label_predict)
-            res = EvaluateResult(num, accuracy, p_r_f[0].tolist(), p_r_f[1].tolist(), p_r_f[2].tolist(), {})
+            uniq_labels = sorted(list(set(label_gold)))
```
Comment (keigohtr) [inline python/sklearn-digits/app.py]: Does it work? I think the label order must not be sorted since we need to correspond them to the output.

**Review by keigohtr — COMMENTED** [review]
Could you check my comment?

File: python/sklearn-digits/app.py (hunk 1 above)
Comment (yuki-mt) [reply python/sklearn-digits/app.py]: That's right. I removed `sorted` , so please check it again.

File: python/sklearn-digits/app.py [hunk 2]
```diff
@@ -97,8 +97,9 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
                 label_predict.append(result.label)
 
             accuracy = accuracy_score(label_gold, label_predict)
-            p_r_f = precision_recall_fscore_support(label_gold, label_predict)
-            res = EvaluateResult(num, accuracy, p_r_f[0].tolist(), p_r_f[1].tolist(), p_r_f[2].tolist(), {})
+            uniq_labels = list(set(label_gold))
```
Comment (keigohtr) [inline python/sklearn-digits/app.py]: Is this label's order correct?

Comment (keigohtr) [reply python/sklearn-digits/app.py]: In this sample, we need to determine the labels by hand to fit the gold label.

Example
```
labels = ["0","1","2","3","4","5","6","7","8","9"]
```

**Review by keigohtr — COMMENTED** [review]
Could you check it again?

File: python/sklearn-digits/app.py (hunk 2 above)
Comment (yuki-mt) [reply python/sklearn-digits/app.py]: @keigohtr 
I explicitly separated label and label_index in https://github.com/rekcurd/rekcurd-example/pull/13/commits/6159e4175a7437a20f1e016570cf14f0d2bf5209 .
( label order is based on `self.labels` )
Please review it

**Review by keigohtr — APPROVED** [review]
Nice work @yuki-mt 
LGTM

**Pull request merged**